src/test_data/arpes.py

```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
     $\k_{perpendicular}$ are the surface parallel and perpendicular momentum,
     $E_{ph}$ is the photon energy, $E_{b}$ is the electron binding energy and
     $\Phi$ is the work function.
+    Parameters
+    ----------
+    photon_energy : float or 1D numpy.ndarray,
+        The photon energy (or energies) in eV.
+    parallel_momentum : float or 1D numpy.ndarray,
+        The parallel momentum in the unit of the inverse of angstrom.
+    binding_energy : float or 1D numpy.ndarray,
+        The binding energy (or energies) in eV.
+
+    inner_potential: float or 1D numpy.ndarray
+        The inner potential in eV.
+    work_function: float or 1D numpy.ndarray
+        The work function in eV.
 
+    Returns
+    -------
+    $k_z$ : float or 1D numpy.ndarray,
```

Comment:
This I think is always a numpy array, never a float. Can you confirm this can be a float or change this?